cmd/state-custom-installer/runBuildInteractive.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
This would be better serviced by an integration test, see for example: https://github.com/ActiveState/cli/blob/820f9e2c27f107c46fc5cf136d52ac2371000927/cmd/state-installer/test/integration/installer_int_test.go#L28